client/dali_grpc_client.py

```diff
@@ -57,18 +66,21 @@ def load_image(img_path: str):
         return np.array(list(img)).astype(np.uint8)
 
 
-def load_images(dir_path: str):
+def load_images(dir_path: str, max_images=-1):
     """
     Loads all files in given dir_path. Treats them as images
     """
+    assert max_images > 0 or max_images == -1, "Please specify sane number of images"
```

Comment:
```suggestion
    assert max_images > 0 or max_images == -1, "max_images must be a positive number or be equal to -1"
```
or
```suggestion
    assert max_images > 0 or max_images == -1
```
the condition is clear enough